packages/select/src/components/multi-select/multiSelect.tsx

```diff
@@ -417,7 +378,17 @@ export class MultiSelect<T> extends AbstractPureComponent<MultiSelectProps<T>, M
             // scroll active item into view after popover transition completes and all dimensions are stable.
             this.queryList.scrollActiveItemIntoView();
         }
-        this.popoverInputForCustomTarget?.focus();
+
+        // Unless user explicitly sets autoFocus to false, for customTargets we should by default
+        // focus on the input on open
```

Comment:
select is doing the same https://github.com/palantir/blueprint/blob/develop/packages/select/src/components/select/select.tsx#L336